service/controller/clusterapi/v30/cloudconfig/cloud_config.go

```diff
@@ -1,153 +1,7 @@
 package cloudconfig
 
-import (
-	"fmt"
-
-	"github.com/giantswarm/microerror"
-	"github.com/giantswarm/micrologger"
-
-	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v30/encrypter"
-)
-
 const (
 	FileOwnerUserName  = "root"
 	FileOwnerGroupName = "root"
 	FilePermission     = 0700
 )
```

Comment:
The constants are the only thing left in this file. We have different implementations for the different stack types TCCP and TCNP.